style.css

```diff
@@ -0,0 +1,240 @@
+* {
+  box-sizing: border-box;
+  padding: 0;
+  margin: 0;
+  text-decoration: none;
+  list-style: none;
+}
+
+body {
+  background-color: #f7f7f9;
+  font-family: poppins, sans-serif;
+}
+
+.header {
+  background-color: white;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 26px 24px;
+}
+
+.container {
+  max-width: 1050px;
+  margin: 0 auto;
+}
+
+.hero-section,
+.about-section {
+  background-color: white;
+  background-image: url("/images/hero-background-image.svg");
```

Comment:
- [ ] Your background image does not fully stretch across the width of the screen. We want our design to follow the provided [Figma template] as close as possible.
<br><br>
**Your design:**
![image](https://user-images.githubusercontent.com/12745474/125136633-206b8180-e103-11eb-923a-336cac96c9c1.png)

**The original design:**
![image](https://user-images.githubusercontent.com/12745474/125136895-8e17ad80-e103-11eb-9d9b-a365506edce8.png)
<br><br>
Let's improve this design by ensuring that that the background image is fully stretched across the screen.

Reply:
![Screenshot from 2021-07-10 17-24-37](https://user-images.githubusercontent.com/83048208/125162898-d4bce480-e1a3-11eb-90e5-ead134a180e8.png)

Here's the screenshot of my screen, It's fully stretched across the width of the screen.